src/tools/material.js

```diff
@@ -18,6 +20,26 @@ function scaleOneLatticeVector(material, key = 'a', factor = 1.0) {
 
 }
 
+function scaleNonPeriodicLattice(material) {
+    const lattice = new Lattice(material.lattice);
+    const basis = new Basis({
+        ...material.basis,
+        cell: lattice.vectorArrays,
+        units: "crystal"
+    })
+    basis.toCartesian();
+    basis.toJSON();
+    //TODO: Change to match the new scaling factor variable from SOF-5214
```

Comment:
I need to merge the code from SOF-5214 before I can update this.

Reply:
Then let's start this branch from code that includes it already??